src/Controls/src/Core/Platform/GestureManager/GestureManager.Windows.cs

```diff
@@ -511,6 +519,81 @@ void OnPointerReleased(object sender, PointerRoutedEventArgs e)
 			PanComplete(true);
 		}
 
+		void OnPgrPointerEntered(object sender, PointerRoutedEventArgs e)
+		{
+			var view = Element as View;
+			if (view == null)
+				return;
+
+			var pointerPoint = e.GetCurrentPoint(Control);
+			var children = (view as IGestureController)?.GetChildElements(new Point(pointerPoint.Position.X, pointerPoint.Position.Y));
+
+			if (children != null)
+				foreach (var recognizer in children.GetChildGesturesFor<PointerGestureRecognizer>())
+				{
+					recognizer.SendPointerEntered(view);
+				}
+
+			if (e.Handled)
+				return;
+
+			IEnumerable<PointerGestureRecognizer> pointerGestures = view.GestureRecognizers.GetGesturesFor<PointerGestureRecognizer>();
```

Comment:
```suggestion
			var pointerGestures = view.GestureRecognizers.GetGesturesFor<PointerGestureRecognizer>();
```

Reply:
done, but curious why?

(I was copying the pattern we have for tapGesture)
```
IEnumerable<TapGestureRecognizer> tapGestures
```

Reply:
it's in our code guidelines to try use var when possible. This is a case tha tis fine how you had, because it's hard to now what is coming from GetGesturesFor.. 
But if was something like `List<PointerGestureRecognizer> pointerGestures = new List<PointerGestureRecognizer>()`,  in that case you shoudl use var because is easy to see what's being created on the right side of the operation.
also having `var ` if you change the type of GetGesturesFor you might not need to go change this code .

Reply:
good to know! thanks Rui :)